src/egon/data/airflow/dags/pipeline.py

```diff
@@ -457,6 +458,7 @@
             data_bundle,
             zensus_mv_grid_districts,
             district_heating_areas,
+            power_plants,
```

Comment:
The HeatSupply-Dataset doesn't have a dependency to the PowerPlants anymore. So this line could be removed.

Reply:
resolved by removing this dependency.